MetaMask/metamask-extension
# Add modal with directions to re-add token as NFT

Adds functionality to pop a modal - which directs them to re-add the asset as an NFT - when a user clicks into the asset details page of an asset we know to be an NFT or clicks the send button of an asset we don't yet identify as an NFT, but upon checking we determine to be one.

**Video demos:**
Scenario 1 - We have previously identified this asset as an ERC721:

https://user-images.githubusercontent.com/34557516/149396538-4fbdf463-90f8-4e71-8920-1c491eb6cb03.mov

Scenario 2 - An ERC1155 (for which we had not previously identified and disabled sending):

https://user-images.githubusercontent.com/34557516/149403470-a42494e5-a5b0-46af-8087-05ac11fc0ed0.mov

(I don't have any ERC1155 tokens that I can use to actually add as a collectible)

Manual testing steps:  

Scenario 1
  - Add an ERC721 as a token (on the Import Token page)
  - Click into the Asset Details page of that token. You should see this modal pop.
  - Click "Yes". You should be redirected to the Import NFT page with the token address pre-populated
 
Scenario 2
 - Add an ERC1155 as a token (on the Import Token page)
  - Click into the Asset Details page of that token. And click the Send button.
  - You should see this modal pop.
  - Click "Yes". You should be redirected to the Import NFT page with the token address pre-populated
